turbinia/config/turbinia_config_tmpl.py

```diff
@@ -73,6 +73,34 @@
 # problems.
 DEBUG_TASKS = False
 
+################################################################################
+#                         External Dependency Configurations
+#
+# Options in this section are used to configure system and docker dependencies.
+################################################################################
+
+# This will allow for the configuration of system dependencies and docker
+# containers. The following configuration will need to be set per job config.
+# Please use the example below as a guidance to adding additional job checks.
+# {
+#   'job': 'The name of the job.'
+#   'programs': 'List of programs to check for.
+#   'docker_image': name of docker image (if any).
+# }
+
+# This will enable the usage of docker containers for the worker.
+DOCKER_ENABLED = False
+
+# Any jobs added to this list will disable it from being used.
+DISABLED_JOBS = []
+
+# Configure additional job dependency checks below.
+DEPENDENCIES = [{
```

Comment:
Even though the other dependencies are covered by the python dependencies and should be installed, I would still include them here to catch other problems.  For example PlasoJob requiring log2timeline.py, etc.  Basically any time we shell out to a specific command in a Task I think we should probably add that to the dependency list here.